app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class GreetingsController < ApplicationController
+      def index
+        @greetings = Greeting.all
+        render json: @greetings
+      end
```

Comment:
- [ ] This function or a function in the react app should return a random message, meaning that every time I refresh the page or click a button, I should get a different message from the API.